src/main/java/com/iota/iri/service/API.java

```diff
@@ -223,8 +223,11 @@ private AbstractResponse process(final String requestString, InetSocketAddress s
                 }
                 case "getBalances": {
                     final List<String> addresses = getParameterAsList(request,"addresses", HASH_SIZE);
+                    final List<String> hashes = request.containsKey("hashes") ?
```

Comment:
here the new field is called `hashes`, but the dto returns `references`.
also the iota.lib.js PR #147 call the new argument `references`, but passes `hashes` to the API.
also this doesn't match `getInclusionStates`'s naming conventions - `tips`

this is confusing - (I ran the API calls a few times w/ `references` till I found this).

Reply:
I'll change it to `tips` in both, then :+1:  ?